src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -816,6 +816,7 @@ def load_arguments(self, _):
         c.argument('managed_image', help='image name(if in the same resource group) or resource id')
         c.argument('os_snapshot', help='Name or ID of OS disk snapshot')
         c.argument('data_snapshots', nargs='+', help='Names or IDs (space-delimited) of data disk snapshots')
+        c.argument('data_snapshot_luns', nargs='+', help='LUNs (space-delimited) of data disk snapshots')
```

Comment:
Maybe we can use the full name of "LUN" for readability, add more description and add the constraints that it should contain the same number of arguments as `--data-snapshots`.

Reply:
to align with `data_snapshots` on meaning, should it be `data_snapshots_luns`?

Reply:
> 
> 
> Maybe we can use the full name of "LUN" for readability, add more description and add the constraints that it should contain the same number of arguments as `--data-snapshots`.

I will add full name of LUN in help.
There is a validation in custom function.

Reply:
> 
> 
> to align with `data_snapshots` on meaning, should it be `data_snapshots_luns`?

I treat "data-snapshot-lun" as an entity, then I add a "s" to represent plural form. I think both names are reasonable.